agents/trainer/task.py

```diff
@@ -113,39 +110,39 @@ def hparams_base():
   """Base hparams tf/Agents PPO """
```

Comment:
Taking another look this will work fine if this is just converted into a dictionary for now.